src/app/app.component.ts

```diff
@@ -124,7 +124,8 @@ export class AppComponent implements OnInit {
     if (omnibarBootstrapConfig) {
       const omnibarConfig: BBOmnibarConfig = {
         serviceName: omnibarBootstrapConfig.serviceName,
-        experimental: omnibarBootstrapConfig.experimental
+        experimental: omnibarBootstrapConfig.experimental,
+        signOutRedirectUrl: omnibarBootstrapConfig.signOutRedirectUrl
```

Comment:
@Blackbaud-JackieKirchhoff: You'll also need to update the `BBOmnibarConfig` interface, located at [`blackbaud/auth-client`](https://github.com/blackbaud/auth-client/blob/master/src/omnibar/omnibar-config.ts).